spk/less/Makefile

```diff
@@ -0,0 +1,23 @@
+SPK_NAME = less
+SPK_VERS = 487
+SPK_REV = 1
+SPK_ICON = src/less.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = less is a pager alternative for the standard unix command called more
+RELOAD_UI = no
+DISPLAY_NAME = less
+STARTABLE = no
+CHANGELOG = "initial version"
+
+HOMEPAGE   = http://www.greenwoodsoftware.com/less/
+LICENSE    = GPLv2
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
```

Comment:
Probably there is no need to refer such a "no-op" script. I have not tested myself, but I guess removing `SSS_SCRIPT` still produces a working package

Reply:
Verified to work without the SSS_SCRIPT setting (and the dsm-control.sh script) on DSM 6.1 / rtd1296.
However, I needed to add cross/termcap to DEPENDS in cross/less/Makefile.